test/e2e/e2e_test.go

```diff
@@ -46,6 +48,8 @@ import (
 	nodeutil "sigs.k8s.io/descheduler/pkg/descheduler/node"
 	podutil "sigs.k8s.io/descheduler/pkg/descheduler/pod"
 	"sigs.k8s.io/descheduler/pkg/descheduler/strategies"
+
+	"sigs.k8s.io/descheduler/pkg/utils"
```

Comment:
Can you put this line together with these `descheduler` imports above?

Reply:
Done